tests/test_connection.py

```diff
@@ -259,3 +259,63 @@ def test_temporary_connection(mock_conn, option, runner):
         schema="PUBLIC",
         warehouse="xsmall",
     )
+
+
+@mock.patch("snowflake.connector.connect")
+@pytest.mark.parametrize("option", ["--temporary-connection", "-x"])
```

Comment:
This test does not need this parameterization. `-x` is the abbreviated form of `--temporary-connection` and both variants of creating temporary connection are already tested in `test_temporary_connection`. You can always use `--temporary-connection` in runner invocation in your test case.